src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -347,6 +358,24 @@ def convert_yaml_to_test(data):
                 new_body["passFailCriteria"]["passFailMetrics"][metric_id][
                     "requestName"
                 ] = name
+    if data.get("autoStop") is not None:
+        if (isinstance(data["autoStop"], str)):
+            new_body["autoStopCriteria"] = {
+                "autoStopDisabled": True,
+            }
+        else:
+            error_rate = data["autoStop"].get("errorPercentage", Decimal(90.0))
```

Comment:
Is there some constant class to manage these constants values at a single places ?

Reply:
will take it up as a refactor task